upack/src/upack/Msg.scala

```diff
@@ -128,7 +128,13 @@ object Obj{
 
   def apply(): Obj = Obj(new mutable.LinkedHashMap[Msg, Msg]())
 }
-case class Ext(tag: Byte, data: Array[Byte]) extends Msg
+case class Ext(tag: Byte, data: Array[Byte]) extends Msg {
+  override def equals(other: Any): Boolean = other match {
```

Comment:
Custom `hashCode()` would be a good idea to cover the rare but possible case that someone uses `Ext` as a Map key.

https://stackoverflow.com/a/2265637

Reply:
Oh bother, of course.

Reply:
Done now.